extractor/src/main/java/org/schabi/newpipe/extractor/exceptions/HttpResponseException.java

```diff
@@ -0,0 +1,15 @@
+package org.schabi.newpipe.extractor.exceptions;
+
+import java.io.IOException;
+import org.schabi.newpipe.extractor.downloader.Response;
+
+public class HttpResponseException extends IOException {

```

Comment:
This should be an `ExtractorException`, since the extractor received an unexpected response. There was no error in data input/output, since the request went through just fine at the network layer.